### PR TITLE
[EN & FR] Home page and corrections

### PR DESCRIPTION
**Pull request by lbourdois**
For French, update of the home page.
For English, correction of two lines

### CODE REVIEW

File: docs/fr/index.md [hunk 1]
```diff
@@ -28,7 +50,34 @@ Consultez le fichier [`README.md`](https://github.com/Atcold/NYU-DLSP21/blob/mas
 - Les éditions précédentes
 
 
+<!--
+## Lectures
+
+Most of the lectures, labs, and notebooks are similar to the previous edition, nevertheless, some are brand new.
+I will try to make clear which is which.
+
+**Legend**: 🖥 slides, 📓 Jupyter notebook, 🎥 YouTube video.
```
Comment (Atcold) [inline docs/fr/index.md:59]: Where's the translation of the legend?

**Review by Atcold — CHANGES_REQUESTED** [review]
Just a little edit. Thanks, though!

**Comment by lbourdois** [comment]
I lacked rigor...  It's corrected :)

**Review by Atcold — APPROVED** [review]

**Comment by Atcold** [comment]
Thanks!

**Pull request merged**